buffer/buffer.go

```diff
@@ -42,6 +42,11 @@ func (b *Buffer) AppendByte(v byte) {
 	b.bs = append(b.bs, v)
 }
 
+// AppendBytes writes a single byte to the Buffer.
```

Comment:
Just came here from the 1.26.0 release notes. Great job! This comment, though, doesn't seem to be right in that the method writes **all** bytes to the buffer.

Reply:
Oops, yes, you're right. I missed this. Thanks!